reposerver/repository/repository.go

```diff
@@ -135,14 +142,14 @@ func (s *Service) runRepoOperation(
 	}
 
 	if source.IsHelm() {
-		helmClient := s.newHelmClient(repo.Repo, repo.GetHelmCreds())
+		version := semver.MustParse(revision)
```

Comment:
The `MustParse` panics if for some reason version cannot be parsed. Please use  `semver.NewVersion`  instead.

Reply:
Good point. I'd assumed that version must be parseable here, because it is today. But code changes to this file could change that assumption.

Reply:
Fixed.